# This is a BitKeeper generated patch for the following project:
# Project Name: Linux kernel tree
# This patch format is intended for GNU patch command version 2.5 or higher.
# This patch includes the following deltas:
#	           ChangeSet	1.1344  -> 1.1345
#	include/linux/toshiba.h	1.3     -> 1.4
#	drivers/char/toshiba.c	1.11    -> 1.12
#
# The following is the BitKeeper ChangeSet Log
# --------------------------------------------
# 03/10/10	noah@caltech.edu	1.1345
# Remove the prototypes for tosh_get_info from include/linux/toshiba.h and
# make the function's definition in drivers/char/toshiba.c static.  This
# function is specific to toshiba.c, so no other file needs the prototype.
#
# This allows drivers/char/toshiba.c to compile with CONFIG_PROC_FS=n and
# allows one to link more than one driver that includes toshiba.h into the
# kernel at the same time without a multiple declaration; for example, both
# drivers/char/toshiba.c and drivers/video/neofb.c.
# --------------------------------------------
#
diff -Nru a/drivers/char/toshiba.c b/drivers/char/toshiba.c
--- a/drivers/char/toshiba.c	Fri Oct 10 18:50:17 2003
+++ b/drivers/char/toshiba.c	Fri Oct 10 18:50:17 2003
@@ -292,7 +292,7 @@
  * Print the information for /proc/toshiba
  */
 #ifdef CONFIG_PROC_FS
-int tosh_get_info(char *buffer, char **start, off_t fpos, int length)
+static int tosh_get_info(char *buffer, char **start, off_t fpos, int length)
 {
 	char *temp;
 	int key;
diff -Nru a/include/linux/toshiba.h b/include/linux/toshiba.h
--- a/include/linux/toshiba.h	Fri Oct 10 18:50:17 2003
+++ b/include/linux/toshiba.h	Fri Oct 10 18:50:17 2003
@@ -33,13 +33,4 @@
 	unsigned int edi __attribute__ ((packed));
 } SMMRegisters;

-#ifdef CONFIG_PROC_FS
-static int tosh_get_info(char *, char **, off_t, int);
-#else /* !CONFIG_PROC_FS */
-inline int tosh_get_info(char *buffer, char **start, off_t fpos, int lenght)
-{
-	return 0;
-}
-#endif /* CONFIG_PROC_FS */
-
 #endif
